### PR TITLE
pep8 and flake8 fixes

### PR DESCRIPTION
**Pull request by bhaveshAn**
did `pep8 / flake8` fixes as in many files `os, sys, json` modules were imported and not used.
Kindly review !

### CODE REVIEW

File: app/scrapers/ask.py [hunk 1]
```diff
@@ -2,14 +2,15 @@
...
+
 class Ask:
     """Scrapper class for Ask"""
     @classmethod
     def __init__(cls):
         pass
 
     @classmethod
-    def get_page(cls,query):
+    def get_ask_page(cls, query):
```
Comment (AnshulMalik) [inline app/scrapers/ask.py]: Why rename?

Comment (AnshulMalik) [reply app/scrapers/ask.py]: Got it

**Comment by bhaveshAn** [comment]
@mariobehling please review !

**Comment by mariobehling** [comment]
Thanks. Please check on Travis.

**Comment by bhaveshAn** [comment]
@mariobehling earlier `travis` was failing due to decorators and cls used in ask.py which are resolved at #205

**Comment by AnshulMalik** [comment]
Redundant with #211. Should be closed

**Comment by mariobehling** [comment]
Thanks. Closing it.

**Pull request closed without merging**